src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const titlesTop = document.querySelector('thead');
+const titlesBottom = document.querySelector('tfoot');
+const table = document.querySelector('tbody');
```

Comment:
```suggestion
const thead = document.querySelector('thead');
const tfoot = document.querySelector('tfoot');
const tbody = document.querySelector('tbody');
```
я думаю так понятнее